WTWSMS/history/characters/lombard.txt

```diff
@@ -82,7 +82,7 @@
 190131 = {
 	name="Appa" #wife of king of Alemanni
 	female=yes
-	dynasty=1044031 #Gausian
+	dynasty=1042348 #Gausian
```

Comment:
What happened here? 1044031 is the Gausian dynasty.

Reply:
There is two dynasties named ''Gausian'': first from vanilla, second from mod.

Reply:
The vanilla one should not be there at all.

Reply:
I'am checked, 1044031 really from vanilla.